packages/navigator/src/browser/navigator-menu.ts

```diff
@@ -7,8 +7,9 @@
 
 import { injectable, inject } from "inversify";
```

Comment:
We should merge the whole file into `navigator-contribution.ts`, it is already responsible for registering menus, keybindings, and commands for the navigator. No need to have other contributions.